### PR TITLE
DI-21389 load ABI

### PR DESCRIPTION
**Pull request by simcha-waintrob**

### CODE REVIEW

File: src/app/relations.json [hunk 1]
```diff
@@ -0,0 +1,87 @@
+[
```
Comment (Chasky-H) [inline src/app/relations.json:1]: What is this?????

Comment (Chasky-H) [reply src/app/relations.json:1]: It only works online now anyways, so why don't you just call /relations

**Comment by simcha-waintrob** [comment]
@Ido ***@***.***> said that the way to get AddonBlocks data is not using relations.
we need to think & decide the right way.
till that we do it using hardcoded data
________________________________

File: src/app/relations.json (hunk 1 above)
Comment (simcha-waintrob) [reply src/app/relations.json:1]: @ido-pepperi said that the way to get AddonBlocks data is not using relations.
we need to think & decide the right way.
till that we do it using hardcoded data

**Review by Chasky-H — APPROVED** [review]

**Pull request merged**